runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsBrothers.java

```diff
@@ -25,25 +25,21 @@
 package net.runelite.client.plugins.barrows;
 
 import lombok.Getter;
-import net.runelite.api.Point;
+import lombok.RequiredArgsConstructor;
+import net.runelite.api.coords.WorldPoint;
 
+@RequiredArgsConstructor
 public enum BarrowsBrothers
 {
-	VERAC("V", new Point(3557, 3298)),
-	DHAROK("D", new Point(3575, 3298)),
-	AHRIM("A", new Point(3566, 3289)),
-	TORAG("T", new Point(3553, 3283)),
-	KARIL("K", new Point(3566, 3275)),
-	GUTHAN("G", new Point(3577, 3283));
+	VERAC("V", new WorldPoint(3557, 3298, 1)),
```

Comment:
This should probably be z=0. And also consider a two arg constructor with default z=0